packages/block-library/src/image/image-size.native.js

```diff
@@ -76,6 +77,14 @@ class ImageSize extends Component {
 			imageWidthWithinContainer: this.state.width,
 			imageHeightWithinContainer: this.state.height,
 		};
+		if ( this.state.width === undefined ) {
+			const width = 300;
+			return (
+				<View onLayout={ this.onLayout } style={ { flex: 1, flexDirection: 'row', justifyContent: 'center' } }>
```

Comment:
WDYT about moving the style definition in a style file? There's already the `./styles.native.scss
` we can use.